pkg/scanner/config.go

```diff
@@ -106,9 +105,8 @@ func GetConfig(configPath string) (*Config, error) {
 		viper.BindEnv("Perceptor.Host")
 		viper.BindEnv("Perceptor.Port")
 
-		viper.BindEnv("Hub.User")
-		viper.BindEnv("Hub.Port")
-		viper.BindEnv("Hub.PasswordEnvVar")
+		viper.BindEnv("BlackDuck.PasswordEnvVar")
```

Comment:
Does this variable contain the name of the environment variable whose values is **just** a password or does the value represent the connection information and should this name of this variable be changed to reflect that?

Reply:
Updated the code